treeAnalysis/caloheader.h

```diff
@@ -124,7 +123,7 @@ float * CalculateM02andWeightedPosition(std::vector<towersStrct> cluster_towers,
         vecTwr += w_i.at(cellI)*TowerPositionVectorFromIndices(cluster_towers.at(cellI).tower_iEta,cluster_towers.at(cellI).tower_iPhi, caloSelect);
     }
     returnVariables[2]=vecTwr.Eta();
-    returnVariables[3]=(vecTwr.Phi()<0 ? vecTwr.Phi()+TMath::Pi() : vecTwr.Phi()-TMath::Pi());
+    returnVariables[3]=vecTwr.Phi();
```

Comment:
I'm a little concerned about this change; I'm not sure why phi was adjusted in the first place.   However, changing it allows calo jets to match properly.

Reply:
I _think_ the idea here was to adjust the phi range, but it looks a bit odd to me. @nschmidtALICE can you comment here?

Reply:
Nico is okay with the change. Merging